lib/rbs/environment.rb

```diff
@@ -288,7 +288,8 @@ def normalize_type_name?(name)
         else
           name
         end
-      end
+
+      type_name?(type_name) ? type_name : false
```

Comment:
```suggestion
      if type_name?(type_name)
        type_name
      end
```

We want `nil` if type name is not present. `false` is for recursive module aliases.